packages/tablecloth-native/tablecloth-native.0.0.2/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+synopsis:
+  "Native OCaml library implementing Tablecloth, a cross-platform standard library for OCaml, Bucklescript and ReasonML"
+description: "Longer description"
+maintainer: "Paul Biggar <paul@darklang.com>"
+authors: "Paul Biggar <paul@darklang.com>"
+license: "MIT with some exceptions"
+homepage: "https://github.com/darklang/tablecloth"
+bug-reports: "https://github.com/darklang/tablecloth/issues"
+depends: [
+  "ocaml"
+  "dune" {build}
+  "base" {>= "v0.10.0"}
+  "str"
```

Comment:
```suggestion
```
There is still a problem. This package doesn't exists. My guess is that you wanted to tell that you are using the `Str` module. However this is a builtin module installed by the compiler.

Reply:
I added this in response to a build failure on this CI. I can't find the log but the error was "No implementations provided for the following modules: Str", and stackoverflow suggested this: https://stackoverflow.com/questions/38547815/no-implementations-provided-for-the-following-modules-str.

I'll remove it and see what happens and we can debug from there.

Thanks again :)

Reply:
str is vendored as a separate library even if it's installed by the compiler. You need to add it explicitly in your dune file for it to be linked

Reply:
Gotcha, thanks! That seems to have fixed it :)